src/style.css

```
@@ -1 +1,127 @@
-/* empty source */
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 32px 16px 16px ;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+}
+
+.card__iMac {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__discription {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  color: #060b35;
+  margin-top: 40px;
+}
+
+.card__productCode {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.reviewSection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.reviewSection--Card-1 {
+  margin-top: 16px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.reviewSection__reviewRate {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.price--card-1 {
+  margin-top: 24px;
+}
+
+.price__text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.price__number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.button__Buy {
```

don't use capital letters if you use international BEM naming convention
```suggestion
.button__buy {
```